### PR TITLE
Updated Readme.md (#161)

### PR DESCRIPTION
**Pull request by rishitachail**
## Describe your changes

1.I updated the Contributors section to add new contributors
2.Added contact with project admin 
using markdown

## Screenshots - If Any (Optional)

README md — C__Users_Rishita Chail_Desktop_GSSOC'23-Projects_Project-Amupedia — Atom 5_22_2023 10_04_21 PM

## Issue ticket number and link - If Any
#161 

## Checklist before requesting a review

- [X ] I have performed a self-review of my code.
- [ X] Starred the repository.

- [ X] Added My Name to the Contributor's List.
- [ X] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [X ] I ran the app and tested it locally to verify that it works as expected.

## Additional Information (Optional)
Any additional information that you want to give us

Please review it and contact me if there is any issue.

### CODE REVIEW

**Comment by rishitachail** [comment]
Okk

File: README.md [hunk 1]
```diff
@@ -88,15 +88,30 @@ Happy coding :)
...
+Show some ❤️ by starring this awesome repository!
+
+
+## 🔗 CONNECT WITH PROJECT ADMIN
+[![portfolio](https://img.shields.io/badge/my_portfolio-000?style=for-the-badge&logo=ko-fi&logoColor=white)](https://github.com/anmode)
+
+[![linkedin](https://img.shields.io/badge/linkedin-0A66C2?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/anmol-agrawal-748389151/)
+
+[![twitter](https://img.shields.io/badge/twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/A2_AnmolAgrawal)
+
```
[the remark this answers is not in the document]
Comment (rishitachail) [reply README.md:115]: README md — C__Users_Rishita Chail_Desktop_GSSOC'23-Projects_Project-Amupedia — Atom 5_22_2023 10_24_44 PM
 
Is it fine?

[the remark this answers is not in the document]
Comment (rishitachail) [reply README.md]: > please align. it! horizontally

@anmode

[the remark this answers is not in the document]
Comment (anmode) [reply README.md]: fine! thankss

**Pull request merged**